mlir/utils/performance/perfRunner.py

```diff
@@ -793,6 +791,161 @@ def benchmarkMIOpenWithMLIRKernels(configs, arch, filename, paths: Paths):
     chip = GFX_CHIP_RE.search(arch).group(0)
     df.to_csv(chip + '_' + filename, index=False)
 
+RUNNABLE_TEST_RE = re.compile(r"//\s*RUN\s*:")
+ROCMLIRGEN_RE = re.compile(r"rocmlir-gen.*-fut")
```

Comment:
You could probably use a match group (something like `rocmlir-gen.*-fut\s*(\w+)`) to pull the name out during the regex match